### PR TITLE
add lilypond visualization

### PR DESCRIPTION
**Pull request by Juphex**
opens a visualization of the notes recognized by the vision node

### CODE REVIEW

File: marimbabot_vision/src/vision_node.py [hunk 1]
```diff
@@ -58,13 +61,14 @@ def callbackImage(data: ROSImage, callback_args):
     except CvBridgeError as e:
       rospy.logerror(e)
 
-    detect_notes(cv_image, pre_processor, model)
+    sentence = detect_notes(cv_image, pre_processor, model)
+    sentence_publisher.publish(sentence)
 
     
-def listener(pre_processor, model):
+def listener(pre_processor, model, sentence_publisher):
```
Comment (Flova) [inline marimbabot_vision/src/vision_node.py:68]: What is the purpose of this function? I think the split is not necessary, and you can integrate it into the other part below.

File: marimbabot_vision/src/vision_node.py [hunk 2]
```diff
@@ -58,13 +61,14 @@ def callbackImage(data: ROSImage, callback_args):
     except CvBridgeError as e:
       rospy.logerror(e)
 
-    detect_notes(cv_image, pre_processor, model)
+    sentence = detect_notes(cv_image, pre_processor, model)
```
[the remark this answers is not in the document]
Comment (Juphex) [reply marimbabot_vision/src/vision_node.py:64]: That is exactly what I have implemented

**Comment by Juphex** [comment]
I need to check whether the picture is shown and then it can be reviewed

File: marimbabot_vision/src/visualization_node.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import numpy as np
+import rospy
+from abjad import Block, LilyPondFile
+from abjad.persist import as_png
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image as ROSImage
+from std_msgs.msg import String
+
+
+def callbackVisionResults(sentence, args):
```
Comment (Flova) [inline marimbabot_vision/src/visualization_node.py:15]: We normally use snake_case, but it is fine imo.

**Review by Flova — APPROVED** [review]
LGTM, if you tested it and it works.

**Comment by Juphex** [comment]
I tested it in Rviz, what is what I got. 
Rviz: 
imageimage

What you sent in our chat @Flova : 
image

**Pull request merged**